### ISSUE TITLE
Diarrohoea for how long? answer options wording not as per DAK

### ISSUE THREAD

**Issue by kaledip** [WorldHealthOrganization/smart-emcare-cmr#28]

**Comment by magp18** [comment]
Yes  I just realized I was using the wrong value set for the Diarrhoea, I have updated it, however for  > 14 days we have EmCare.B10S1.DE08 which is More than 14 days but Diarrhoea for how long? has EmCare.B11S1.DE04 - 14 days or More. I was recycling the first one for the Diarrhoea as well since it's mentioned in other value sets but I can change it back to EmCare.B10S1.DE08 - More than 14. Let me know what you think it's best, thanks!
